test/index.html

```diff
@@ -1,97 +0,0 @@
-<!DOCTYPE HTML>
```

Comment:
it would be nice to still have a test/index.html file we can use to load in the tests manually in the browser.